src/components/full-view-accommodation/full-view-accommodation.tsx

```diff
@@ -66,41 +128,62 @@ export default function ShowPhotos({ id, open, onClose }: ShowPhotosProps) {
           </Typography>
         </Toolbar>
 
-        <Grid sx={{ height: '100%', margin: '0 auto' }}>
+        <Grid
```

Comment:
I am not sure if grid is a good choice to have a custom style, it might be replaced with Box or Stack